scripts/MultiTester.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const {RuleTester} = require('eslint');
+
+/**
+ * Wrapper for ESLint's RuleTester class that runs tests against both the
+ * standard ("espree") parser and the `@typescript-eslint/parser` one.
+ */
+class MultiTester extends RuleTester {
+	constructor(options) {
+		super(options);
+
+		this._liferay = {
```

Comment:
Using this "namespace" here to avoid possibly clobbering a current or future property with the name `_parsers`.